frontend/app/styles/app.scss

```diff
@@ -192,6 +193,21 @@ li a {
   margin-left: 6px;
 }
 
+.encryptable-row-icons {
+  margin-left: auto;
+}
+
+@media (max-width: 1062px) {
+  .encryptable-row-attribute,
+  .encryptable-row-copy-button {
+    display: none !important;
```

Comment:
important really needed?
and why do we hide this button if the screen is small?

Reply:
Class "ecryptable-row-icons" has been removed and reimplemented via Bootstrap. The media query stays because we have to hide the button otherwise the row wouldn't fit properly on the screen.